azure-mgmt-eventgrid/azure/mgmt/eventgrid/models/__init__.py

```diff
@@ -33,7 +33,6 @@
 from .topic_type_info_paged import TopicTypeInfoPaged
 from .event_grid_management_client_enums import (
     EventSubscriptionProvisioningState,
-    OperationOrigin,
```

Comment:
I don't see removal of this file in the PR, you probably regenerated in place. Could you erase your azure-mgmt-eventgrid/azure/mgmt/eventgrid/ folder and regenerate?

Also, please add removal of this enum in the ChangeLog

Reply:
Nope, that's an enum, no file removal. But still would appreciate if you can add that in the ChangeLog :)

Reply:
Done, thanks!